Makefile

```diff
@@ -34,30 +34,58 @@ build-all:
 build-release:
 	CARGO_PROFILE_RELEASE_LTO=true RUSTFLAGS="-C codegen-units=1" cargo build --locked --features with-all-runtime --release --workspace --exclude runtime-integration-tests --exclude e2e-tests --exclude test-service
 
+.PHONY: build-production
+build-production:
```

Comment:
we don't need both production, release, and internal release
we can just update the release one to production

Reply:
do we still need `internal release` or we can replace `release` and `internal release` all to `production`? some docker script rely on `internal release`. i.e. docker-hub-publish-acala-dev.sh use `internal release`, while docker-hub-publish-acala-dev.sh use `release`.

Reply:
replace `release` with `production`, and keep `internal release`.